docker/rasterize-chrome/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM demisto/python3-deb:3.7.4.1163
+
+COPY requirements.txt .
+COPY google-chrome-stable_current_amd64.deb .
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  gcc \
+  python-dev \
+  wget \
+  libxss1 \
+  libappindicator1 \
+  libindicator7 \
+&& pip3 install --no-cache-dir -r requirements.txt
+
+# Install Chrome
+#RUN wget https://dl.google.co/linux/direct/google-chrome-stable_current_amd64.deb; \
+RUN dpkg -i google-chrome*.deb; \
```

Comment:
Use `&&` and not `;`

Reply:
&& will not run the next command on an exit code other than 0